docs/markdown/Docker/docker.md

```diff
@@ -171,7 +181,7 @@ very-secret-value
 
 ### Build Docker image example
 
-This example copies both a `file` and `pex_binary`:
+This example copies both a `file` and `pex_binary`. The file is specified as an explicit dependency in the `BUILD` file, whereas the `pex_binary` dependency is inferred from the path in the `Dockerfile`.
```

Comment:
Well that should perhaps be the case, but right now, I see an explicit dependency also on the `pex_binary`..

Reply:
Removed and tested and it works without.